GoogleForCreators/web-stories-wp
# Fix image cropping for non-admins

## Context



Users with editor role were not able to crop poster images

## Summary



Fixes an issue with image cropping for users without the `customize` capability.

Directly using `CustomizeImageCropper` will set the `wp_customize` param which only works for admins, but in our case the cropping should also work for non-admins.

## Relevant Technical Choices



Adds a custom cropper similar to `CustomizeImageCropper`.

## To-do



## User-facing changes



N/A

## Testing Instructions






This PR can be tested by following these steps:

1. Log in as an editor
2. Create a new story
3. Try to set a poster image or publisher logo which requires cropping
4. Verify that cropping works.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8912
